pandas/tests/tseries/offsets/test_business_day.py

```diff
@@ -30,7 +30,7 @@
 
 
 class TestBusinessDay(Base):
-    _offset = BDay
+    _offset: type[BDay] = BDay
```

Comment:
This and some other sub-classes of `Base` require more concrete sub-classes for `_offset`.

This will trigger mypy errors once offset.pyi is added.